utils/keyEvents.js

```diff
@@ -4,12 +4,7 @@ const _inputKeyPressDown = (e, value, setOpen, formatTimeValue, setFocus) => {
     case 9: // tab
     case 13: // return
       setOpen(false);
-      if (value) {
-        // formatValue
-        // hide list
-        // basically same as when clicking on blur
-        formatTimeValue(value);
-      }
+      formatTimeValue(value);
```

Comment:
If there is a default value and the user clicks into the input, clears the value and then tabs out of the field; the value doesn't update.  I think we can safely remove this for the tab, return and esc keys cause if the input loses focus then what is in the input should be the value.  Also, validation will run at the end of the call stack which is kind of a catch all if something strange happens.